### ISSUE TITLE
Input inside iframe sometimes not work

### ISSUE THREAD

**Issue by pedro3900** [driftyco/ionic#7219]
I am using ionic with ios 9 and iframe. In my aplication I use cordova-plugin-inappbrowser and cordova-plugin-keyboard.

When I click in an input inside my iframe if I click more than one time keyboard appear but when I click in any letter not write nothing, if I click in other input or in this input only one time works fine.

If I go to the website directly with the browser safari all go fine. 

This error only appear in ios, in android all works fine.

I am using ionic driftyco/ionic-bower#1.2.1

**Comment by jgw96** [comment]
Hello, thanks for opening an issue with us! Do you have a repo that i can reproduce this with?

**Comment by pedro3900** [comment]
No I haven´t a repo with it :(

**Comment by jgw96** [comment]
Unfortunately, without a minimal test case there is not much i can do for this issue. If you could provide a plunker, codepen or repo that i can use to reproduce this issue that would be great! Thanks!

**Comment by pedro3900** [comment]
ok, thanks, I put the issue to see if somebody has the same problem

**Comment by jgw96** [comment]
Hello all! As it seems it has been a while since there was any activity on this issue i will be closing it for now. Feel free to comment if you are still running into this issue. Thanks for using Ionic!

**Comment by adamduren** [comment]
I am experiencing what I believe to be the same or similar issue. In my instance it's caused by trying to use the Stripe Elements library in my Ionic app. You pass the library an element for it to instantiate itself into. It embed an iframe into the element you passed which has inputs. This is where the problem begins.

I have recreated the issue from the `ionic-tabs` starter using a method similar to what I described above for reproduction.

https://github.com/adamduren/ionic-keyboard-iframe-bug

**GIF illustrating issue attached below**

**Description of issues**
The primary issue is that clicking outside of the iframe does not dismiss the keyboard and there is no way to dismiss the keyboard without some workaround.

The secondary issue caused by the workaround of detecting the blur on the iframe and manually dismissing the keyboard is that on the next input focus the keyboard does not take into account where the input is and then covers it up.

![output](https://user-images.githubusercontent.com/581097/33887172-e681423a-df16-11e7-9569-47600f6f4d3b.gif)
